homeassistant/components/yolink/translations/en.json

```diff
@@ -1,20 +1,25 @@
 {
     "config": {
-        "abort": {
-            "already_configured": "Account is already configured",
-            "already_in_progress": "Configuration flow is already in progress",
-            "authorize_url_timeout": "Timeout generating authorize URL.",
-            "missing_configuration": "The component is not configured. Please follow the documentation.",
-            "no_url_available": "No URL available. For information about this error, [check the help section]({docs_url})",
-            "oauth_error": "Received invalid token data."
+      "step": {
+        "pick_implementation": {
+          "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]"
         },
-        "create_entry": {
-            "default": "Successfully authenticated"
-        },
-        "step": {
-            "pick_implementation": {
-                "title": "Pick Authentication Method"
-            }
+        "reauth_confirm": {
+          "title": "[%key:common::config_flow::title::reauth%]",
+          "description": "The yolink integration needs to re-authenticate your account"
         }
+      },
+      "abort": {
+        "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
+        "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
+        "oauth_error": "[%key:common::config_flow::abort::oauth2_error%]",
+        "missing_configuration": "[%key:common::config_flow::abort::oauth2_missing_configuration%]",
+        "authorize_url_timeout": "[%key:common::config_flow::abort::oauth2_authorize_url_timeout%]",
+        "no_url_available": "[%key:common::config_flow::abort::oauth2_no_url_available%]",
+        "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
+      },
+      "create_entry": {
+        "default": "[%key:common::config_flow::create_entry::authenticated%]"
+      }
     }
-}
\ No newline at end of file
+  }
```

Comment:
Don't update this file manually. Run the translations develop script after updating the `strings.json` file. The `strings.json` file is the source of truth.

https://developers.home-assistant.io/docs/internationalization/core#test-translations